Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py

```diff
@@ -0,0 +1,192 @@
+import os
+import traceback
+
+import demistomock as demisto  # noqa: F401
+import requests
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable warning for insecure requests when cert validation is disabled
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Get dict of CT instance parameters and command args
+CT_PARAMS = demisto.params()
+CMD_ARGS = demisto.args()
+
+SCAN_OPTIONS = ['pr', 'nw', 'nc', 'st', 'sc', 'ru', 'co', 'lo', 'ns', 'wb', 'fs']
+CT_SERVER = CT_PARAMS['server']
+REST_PORT = CT_PARAMS['rest_port']
+API_KEY = CT_PARAMS['api_key']
+USER = CT_PARAMS['credentials']['identifier']
+PASSWORD = CT_PARAMS['credentials']['password']
+VERIFY_SERVER_CERT = False
+REQ_HEADERS = {'restApiKey': API_KEY}
+USE_PROXY = CT_PARAMS['use_proxy']
+
+
+def IS_2XX(x):
```

Comment:
Why isn't this method called `is_2xx` (lowercase)?
No need to decorate it with `@staticmethod`?

Reply:
It's just a function - not a method of a class - therefore no need.